Documentation/Fluid/Index.rst

```diff
@@ -22,40 +22,41 @@ projects without TYPO3.
 Should I use Fluid?
 ---------------------
 
-Fluid is the de-facto-standard for any HTML based output in TYPO3. It's used by
-Extbase-Extensions by default and is available to TypoScript via
+Fluid is the de-facto standard for any HTML based output in TYPO3. It's used by
+Extbase Extensions by default and is available in TypoScript via
```

Comment:
Since "extension" does not belong to the term / definition of "Extbase", I would recommend to write it lowercase:
"Extbase extensions"